compiler/one-cmds/one-build.cfg

```diff
@@ -0,0 +1,20 @@
+[driver-to-run]
```

Comment:
How about naming this filas as `one-build.cfg.template` ?

Reply:
@seanshpark How about `one-build.template.cfg`? Editor programs like vim or sth recognize its format as a configuration  file from last extension. 
`one-build.cfg.template`
![image](https://user-images.githubusercontent.com/24587354/97241824-8c3be180-1835-11eb-96e1-bf5cab5e7733.png)
`one-build.template.cfg`
![image](https://user-images.githubusercontent.com/24587354/97241858-a83f8300-1835-11eb-8ad4-0afcc0b3451e.png)